src/class-wp-curate.php

```diff
@@ -12,4 +12,32 @@
  */
 class WP_Curate {
 
+	/**
+	 * Get the query heading based on the curation settings.
+	 *
+	 * @param array<mixed> $curation Curation settings.
+	 * @return string Query heading.
+	 */
+	public static function get_query_heading( array $curation ): string {
+		$heading  = '';
+		$provider = $curation[ 'provider' ];
+
+		if ( taxonomy_exists( $provider ) && isset( $curation[ $provider ] ) && is_numeric( $curation[ $provider ] ) ) { // @phpstan-ignore-line
+			$term = get_term( $curation[ $provider ], $provider ); // @phpstan-ignore-line
```

Comment:
I was able to not ignore this line by running `intval()` on `$curation[ $provider ]`